### ISSUE TITLE
False positive -> node-sass

### ISSUE THREAD

**Issue by siddhantvirus** [depcheck/depcheck#605]
### Bug Description

depcheck says that [node-sass](https://www.npmjs.com/package/node-sass) is an unused dependency but it's required to use any ```.scss``` files  in a project, and does not require the node-sass package to be imported anywhere for the same.
### Code snippets

Where the dependency is listed in package.json: dependencies

```
"node-sass": "^4.14.1",
```
### Versions

- `node -v`: v14.3.0
- `npm -v`: 6.14.8

**Comment by siddhantvirus** [comment]
Could anyone help me figure how could I try and fix this ?

**Comment by rumpl** [comment]
Hey @siddhantvirus 

Could you paste the output of `depcheck --json` here please?

**Comment by siddhantvirus** [comment]
Hi @rumpl , I'm sorry for not being able o attach the same earlier . here is the output of `npx depcheck --json`

```json
{
    "dependencies": [
        "@material-ui/icons",
        "@testing-library/user-event",
        "node-sass",
        "react-cookie",
        "react-map-gl",
        "react-mapbox-gl",
        "react-otp-timer",
        "react-phone-number-input",
        "react-tooltip",
        "semantic-ui-css",
        "swipe-js-iso",
        "universal-cookie"
    ],
    "devDependencies": [],
    "missing": {
        "eslint-config-react-app": [
            "<path of project>\\package.json"
        ],
        "babel-eslint": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-import": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-flowtype": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-jsx-a11y": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-react": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-react-hooks": [
            "<path of project>\\package.json"
        ],
        "@fortawesome/fontawesome-free": [
            "<path of project>\\src\\App.js"
        ],
        "history": [
            "<path of project>\\src\\history.js"
        ],
        "bootstrap-css-only": [
            "<path of project>\\src\\index.js"
        ],
        "screen": [
            "<path of project>\\src\\components\\OrderHistoryContainer\\index.scss"
        ],
        "only": [
            "<path of project>\\src\\components\\MobileSearch\\MobileSearch.scss",
            "<path of project>\\src\\components\\Error\\Error.scss"
        ],
        "prop-types": [
            "<path of project>\\src\\components\\Body-Home\\DraggableMarker.js"
        ]
    },
    "using": {
        "react-scripts": [
            "<path of project>\\package.json"
        ],
        "eslint-config-react-app": [
            "<path of project>\\package.json"
        ],
        "babel-eslint": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-import": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-flowtype": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-jsx-a11y": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-react": [
            "<path of project>\\package.json"
        ],
        "eslint-plugin-react-hooks": [
            "<path of project>\\package.json"
        ],
        "react": [
            "<path of project>\\src\\App.js",
            "<path of project>\\src\\App.test.js",
            "<path of project>\\src\\ConfirmPass.js",
            "<path of project>\\src\\Dashboard.js",
            "<path of project>\\src\\index.js",
            "<path of project>\\src\\LoginForm.jsx",
            "<path of project>\\src\\LoginHeader.jsx",
            "<path of project>\\src\\OtpAuth.jsx",
            "<path of project>\\src\\Register.js",
            "<path of project>\\src\\components\\AuthApi.jsx",
            "<path of project>\\src\\components\\ProtectedRoute.jsx",
            "<path of project>\\src\\components\\Session.jsx",
            "<path of project>\\src\\components\\userprofile\\favourites.jsx",
            "<path of project>\\src\\components\\userprofile\\notifications.jsx",
            "<path of project>\\src\\components\\userprofile\\usercommunity.jsx",
            "<path of project>\\src\\components\\userprofile\\userprofile.jsx",
            "<path of project>\\src\\components\\TermsAndCondition\\index.jsx",
            "<path of project>\\src\\components\\StockArea\\AddStockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockAreaDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\AddProduct.jsx",
            "<path of project>\\src\\components\\ProductInfo\\CarouselImage.js",
            "<path of project>\\src\\components\\ProductInfo\\Counter.js",
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfo.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfoDetail.jsx",
            "<path of project>\\src\\components\\OrderHistoryContainer\\index.jsx",
            "<path of project>\\src\\components\\OrderHistoryContainer\\components\\OrderItem\\index.jsx",
            "<path of project>\\src\\components\\OrderHistoryContainer\\components\\ItemSold\\index.jsx",
            "<path of project>\\src\\components\\OrderHistoryContainer\\components\\ItemBought\\index.jsx",
            "<path of project>\\src\\components\\OrderHistory\\OrderHistCard.js",
            "<path of project>\\src\\components\\OrderHistory\\orderHistory.jsx",
            "<path of project>\\src\\components\\orderDetailsPopup\\orderDetailsPopup.jsx",
            "<path of project>\\src\\components\\OrderDetails\\index.jsx",
            "<path of project>\\src\\components\\NotFound\\NotFound.js",
            "<path of project>\\src\\components\\newSearch\\NewSearch.jsx",
            "<path of project>\\src\\components\\MobileSearch\\MobileSearch.jsx",
            "<path of project>\\src\\components\\Map\\Map.js",
            "<path of project>\\src\\components\\Map\\Popup.js",
            "<path of project>\\src\\components\\Login-Signup\\newLogin.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newSignUp.jsx",
            "<path of project>\\src\\components\\Header\\InfoHeader.jsx",
            "<path of project>\\src\\components\\Header\\InfoHeaderMain.jsx",
            "<path of project>\\src\\components\\Header\\Navigation.jsx",
            "<path of project>\\src\\components\\Header\\Toggler.jsx",
            "<path of project>\\src\\components\\Footer\\Footer.jsx",
            "<path of project>\\src\\components\\Error\\Error.jsx",
            "<path of project>\\src\\components\\ContactUs\\ContactUs.js",
            "<path of project>\\src\\components\\Community_Single\\Community_Single.js",
            "<path of project>\\src\\components\\Community_Single\\Comm_Product.js",
            "<path of project>\\src\\components\\Community\\Addcommunity.jsx",
            "<path of project>\\src\\components\\Community\\Community.js",
            "<path of project>\\src\\components\\Community\\CommunityCard.js",
            "<path of project>\\src\\components\\buyProduct\\buyProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddStockArea.jsx",
            "<path of project>\\src\\components\\Body-Home\\Carousel.jsx",
            "<path of project>\\src\\components\\Body-Home\\DraggableMarker.js",
            "<path of project>\\src\\components\\Body-Home\\MapModal.js",
            "<path of project>\\src\\components\\Body-Home\\Recentbox.js",
            "<path of project>\\src\\components\\Body-Home\\Search.jsx"
        ],
        "react-router-dom": [
            "<path of project>\\src\\App.js",
            "<path of project>\\src\\ConfirmPass.js",
            "<path of project>\\src\\index.js",
            "<path of project>\\src\\LoginForm.jsx",
            "<path of project>\\src\\LoginHeader.jsx",
            "<path of project>\\src\\Register.js",
            "<path of project>\\src\\components\\ProtectedRoute.jsx",
            "<path of project>\\src\\components\\userprofile\\usercommunity.jsx",
            "<path of project>\\src\\components\\userprofile\\userprofile.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfoDetail.jsx",
            "<path of project>\\src\\components\\OrderHistory\\OrderHistCard.js",
            "<path of project>\\src\\components\\OrderHistory\\orderHistory.jsx",
            "<path of project>\\src\\components\\newSearch\\NewSearch.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newLogin.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newSignUp.jsx",
            "<path of project>\\src\\components\\Header\\InfoHeader.jsx",
            "<path of project>\\src\\components\\Header\\InfoHeaderMain.jsx",
            "<path of project>\\src\\components\\Header\\Toggler.jsx",
            "<path of project>\\src\\components\\Footer\\Footer.jsx",
            "<path of project>\\src\\components\\Community_Single\\Community_Single.js",
            "<path of project>\\src\\components\\Community\\Addcommunity.jsx",
            "<path of project>\\src\\components\\Community\\CommunityCard.js"
        ],
        "@fortawesome/fontawesome-free": [
            "<path of project>\\src\\App.js"
        ],
        "axios": [
            "<path of project>\\src\\App.js",
            "<path of project>\\src\\ConfirmPass.js",
            "<path of project>\\src\\Dashboard.js",
            "<path of project>\\src\\LoginForm.jsx",
            "<path of project>\\src\\LoginHeader.jsx",
            "<path of project>\\src\\OtpAuth.jsx",
            "<path of project>\\src\\Register.js",
            "<path of project>\\src\\components\\ProtectedRoute.jsx",
            "<path of project>\\src\\components\\userprofile\\favourites.jsx",
            "<path of project>\\src\\components\\userprofile\\notifications.jsx",
            "<path of project>\\src\\components\\userprofile\\usercommunity.jsx",
            "<path of project>\\src\\components\\userprofile\\userprofile.jsx",
            "<path of project>\\src\\components\\StockArea\\AddStockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockArea.jsx",
            "<path of project>\\src\\components\\ProductInfo\\AddProduct.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfo.jsx",
            "<path of project>\\src\\components\\OrderHistory\\orderHistory.jsx",
            "<path of project>\\src\\components\\Map\\Map.js",
            "<path of project>\\src\\components\\Login-Signup\\newLogin.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newSignUp.jsx",
            "<path of project>\\src\\components\\ContactUs\\ContactUs.js",
            "<path of project>\\src\\components\\Community_Single\\Community_Single.js",
            "<path of project>\\src\\components\\Community\\Addcommunity.jsx",
            "<path of project>\\src\\components\\Community\\Community.js",
            "<path of project>\\src\\components\\Community\\CommunityCard.js",
            "<path of project>\\src\\components\\Body-Home\\AddProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddStockArea.jsx",
            "<path of project>\\src\\components\\Body-Home\\Recentbox.js",
            "<path of project>\\src\\components\\Body-Home\\Search.jsx"
        ],
        "@testing-library/react": [
            "<path of project>\\src\\App.test.js"
        ],
        "react-dom": [
            "<path of project>\\src\\App.test.js",
            "<path of project>\\src\\ConfirmPass.js",
            "<path of project>\\src\\index.js",
            "<path of project>\\src\\LoginForm.jsx",
            "<path of project>\\src\\LoginHeader.jsx",
            "<path of project>\\src\\OtpAuth.jsx",
            "<path of project>\\src\\Register.js",
            "<path of project>\\src\\components\\userprofile\\userprofile.jsx",
            "<path of project>\\src\\components\\StockArea\\AddStockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockAreaDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\AddProduct.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfoDetail.jsx",
            "<path of project>\\src\\components\\OrderHistoryContainer\\index.jsx",
            "<path of project>\\src\\components\\MobileSearch\\MobileSearch.jsx",
            "<path of project>\\src\\components\\Map\\Map.js",
            "<path of project>\\src\\components\\Login-Signup\\newLogin.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newSignUp.jsx",
            "<path of project>\\src\\components\\Error\\Error.jsx",
            "<path of project>\\src\\components\\Community\\Addcommunity.jsx",
            "<path of project>\\src\\components\\Community\\CommunityCard.js",
            "<path of project>\\src\\components\\Body-Home\\AddProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddStockArea.jsx",
            "<path of project>\\src\\components\\Body-Home\\MapModal.js",
            "<path of project>\\src\\components\\Body-Home\\Search.jsx"
        ],
        "@material-ui/core": [
            "<path of project>\\src\\ConfirmPass.js",
            "<path of project>\\src\\LoginHeader.jsx",
            "<path of project>\\src\\OtpAuth.jsx",
            "<path of project>\\src\\Register.js",
            "<path of project>\\src\\components\\OrderHistoryContainer\\index.jsx",
            "<path of project>\\src\\components\\MobileSearch\\MobileSearch.jsx",
            "<path of project>\\src\\components\\Error\\Error.jsx"
        ],
        "firebase": [
            "<path of project>\\src\\ConfirmPass.js"
        ],
        "jquery": [
            "<path of project>\\src\\Dashboard.js",
            "<path of project>\\src\\OtpAuth.jsx",
            "<path of project>\\src\\Register.js",
            "<path of project>\\src\\components\\StockArea\\AddStockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockAreaDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\AddProduct.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfo.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfoDetail.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newSignUp.jsx",
            "<path of project>\\src\\components\\Header\\Navigation.jsx",
            "<path of project>\\src\\components\\Community\\Addcommunity.jsx",
            "<path of project>\\src\\components\\Community\\Community.js",
            "<path of project>\\src\\components\\Body-Home\\AddProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddStockArea.jsx",
            "<path of project>\\src\\components\\Body-Home\\Search.jsx"
        ],
        "history": [
            "<path of project>\\src\\history.js"
        ],
        "font-awesome": [
            "<path of project>\\src\\index.js"
        ],
        "bootstrap-css-only": [
            "<path of project>\\src\\index.js"
        ],
        "mdbreact": [
            "<path of project>\\src\\index.js"
        ],
        "react-google-login": [
            "<path of project>\\src\\LoginForm.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newLogin.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newSignUp.jsx"
        ],
        "js-cookie": [
            "<path of project>\\src\\LoginForm.jsx",
            "<path of project>\\src\\components\\Session.jsx"
        ],
        "react-modal": [
            "<path of project>\\src\\LoginForm.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newLogin.jsx"
        ],
        "bootstrap": [
            "<path of project>\\src\\Register.js",
            "<path of project>\\src\\components\\StockArea\\AddStockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockAreaDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\AddProduct.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfo.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfoDetail.jsx",
            "<path of project>\\src\\components\\Community\\Addcommunity.jsx",
            "<path of project>\\src\\components\\Community\\Community.js",
            "<path of project>\\src\\components\\Body-Home\\AddProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddStockArea.jsx",
            "<path of project>\\src\\components\\Body-Home\\Search.jsx"
        ],
        "@testing-library/jest-dom": [
            "<path of project>\\src\\setupTests.js"
        ],
        "react-bootstrap": [
            "<path of project>\\src\\components\\userprofile\\userprofile.jsx",
            "<path of project>\\src\\components\\StockArea\\AddStockArea.jsx",
            "<path of project>\\src\\components\\ProductInfo\\AddProduct.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx",
            "<path of project>\\src\\components\\ProductInfo\\ProductInfoDetail.jsx",
            "<path of project>\\src\\components\\Community\\Addcommunity.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddStockArea.jsx",
            "<path of project>\\src\\components\\Body-Home\\Search.jsx"
        ],
        "mapbox-gl": [
            "<path of project>\\src\\components\\StockArea\\AddStockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockArea.jsx",
            "<path of project>\\src\\components\\Map\\Map.js",
            "<path of project>\\src\\components\\Body-Home\\AddProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddStockArea.jsx",
            "<path of project>\\src\\components\\Body-Home\\DraggableMarker.js",
            "<path of project>\\src\\components\\Body-Home\\Search.jsx"
        ],
        "react-swipe": [
            "<path of project>\\src\\components\\StockArea\\StockArea.jsx",
            "<path of project>\\src\\components\\StockArea\\StockAreaDetail.jsx"
        ],
        "react-responsive-carousel": [
            "<path of project>\\src\\components\\ProductInfo\\CarouselImage.js"
        ],
        "@chakra-ui/core": [
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx"
        ],
        "@emotion/core": [
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx"
        ],
        "@emotion/styled": [
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx"
        ],
        "emotion-theming": [
            "<path of project>\\src\\components\\ProductInfo\\ProductDetail.jsx"
        ],
        "screen": [
            "<path of project>\\src\\components\\OrderHistoryContainer\\index.scss"
        ],
        "only": [
            "<path of project>\\src\\components\\MobileSearch\\MobileSearch.scss",
            "<path of project>\\src\\components\\Error\\Error.scss"
        ],
        "semantic-ui-react": [
            "<path of project>\\src\\components\\Map\\Map.js"
        ],
        "aos": [
            "<path of project>\\src\\components\\Login-Signup\\newLogin.jsx",
            "<path of project>\\src\\components\\Login-Signup\\newSignUp.jsx"
        ],
        "react-inputs-validation": [
            "<path of project>\\src\\components\\Community\\Addcommunity.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddStockArea.jsx",
            "<path of project>\\src\\components\\Body-Home\\Search.jsx"
        ],
        "react-router": [
            "<path of project>\\src\\components\\Community\\CommunityCard.js"
        ],
        "react-lines-ellipsis": [
            "<path of project>\\src\\components\\Community\\CommunityCard.js"
        ],
        "reactjs-tag-input": [
            "<path of project>\\src\\components\\Body-Home\\AddProduct.jsx",
            "<path of project>\\src\\components\\Body-Home\\AddStockArea.jsx",
            "<path of project>\\src\\components\\Body-Home\\Search.jsx"
        ],
        "prop-types": [
            "<path of project>\\src\\components\\Body-Home\\DraggableMarker.js"
        ],
        "react-google-maps": [
            "<path of project>\\src\\components\\Body-Home\\MapModal.js"
        ],
        "react-google-autocomplete": [
            "<path of project>\\src\\components\\Body-Home\\MapModal.js"
        ],
        "react-geocode": [
            "<path of project>\\src\\components\\Body-Home\\MapModal.js"
        ]
    },
    "invalidFiles": {},
    "invalidDirs": {}
}
```

**Comment by mr-feek** [comment]
I just ran into the same thing but with `vue-template-compiler`. The readme says to install it, then the tool complains that it is installed and not being used

**Comment by siddhantvirus** [comment]
@rumpl could look into this issue or recommend a fix that one could implement?

**Comment by sepsol** [comment]
I also have the same issue on a `create-react-app` project.
